src/app/screens/sendStx/stxAmountSelector.tsx

```diff
@@ -0,0 +1,250 @@
+import useWalletSelector from '@hooks/useWalletSelector';
```

Comment:
There's a lot of duplication in this component and the btc one. Maybe we could extract a shared shell component which takes in values and functions which power the view and make btc and stx specific implementations which wrap the shell. It will be useful for reuse on brc-20, sip-10 and rune screens.